apps/trivium/src/kreyvium/test.rs

```diff
@@ -0,0 +1,377 @@
+use tfhe::prelude::*;
+use tfhe::{generate_keys, ConfigBuilder, FheBool, FheUint64, FheUint8};
+
+use crate::{KreyviumStream, KreyviumStreamByte, KreyviumStreamShortint, TransCiphering};
+
+///! Values for these tests come from: https://github.com/renaud1239/Kreyvium
```

Comment:
same remark for the doc comment, and same thing to get a given commit to avoid having that link break if possible